packages/ndla-ui/src/ContactBlock/ContactBlock.tsx

```diff
@@ -150,9 +150,17 @@ interface LinkWrapperProps {
   children: ReactNode;
 }
 
+const StyledLink = styled.a`
+  text-decoration: none;
+`;
```

Comment:
Gjør dette kun for å droppe default anchor-styling. Ser heller nærmere på det når ContactBlock skal refaktoreres.